tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "@tsconfig/react-native/tsconfig.json",
+  "compilerOptions": {
+    "target": "es2018",
```

Comment:
we should target es2022

Reply:
how we select? assuming BE uses most modern browsers and mobile support it?

Reply:
we target es2022 in hydra as well (it's the most readable output so preferable for security audits). The platforms transpile any syntax that's not supported anyway